emily/handler/src/api/handlers/internal.rs

```diff
@@ -139,10 +141,12 @@ pub async fn execute_reorg_handler(
             entry.reorganize_around(&request.canonical_tip)?;
             match accessors::set_deposit_entry(context, &mut entry).await {
                 Ok(_) => break,
-                Err(Error::VersionConflict) => {
-                    debug!(
-                        "Encountered race condition in updating entry {:?}. Attempt {}/{}",
-                        entry, attempt, ENTRY_UPDATE_RETRIES
+                Err(Error::VersionConflict(error)) => {
+                    warn!(
+                        %error,
+                        ?entry,
+                        %attempt,
+                        "Encountered race condition in updating entry, max attempts: {ENTRY_UPDATE_RETRIES}",
```

Comment:
I'd keep this out of the interpolation as well to keep the messages static (i.e. aggregate-able regardless of the value).

Reply:
Fixed a bunch in https://github.com/stacks-sbtc/sbtc/pull/1627/commits/b12fc4c4c52c0e531dc6bb7e356eb3a11444cb69